temporian/implementation/numpy/data/sampling.py

```diff
@@ -0,0 +1,9 @@
+from typing import Dict, List, Tuple
+
+import numpy as np
+
+
+class NumpySampling:
+    def __init__(self, names: List[str], data: Dict[Tuple, np.ndarray]) -> None:
```

Comment:
Is this like the "index" in NumpyFeature?

Reply:
It is - I'll update `NumpyFeature` to have this instead of `index`.